src/picongpu/include/plugins/IsaacPlugin.hpp

```diff
@@ -400,12 +403,10 @@ class IsaacPlugin : public ILightweightPlugin
                 json_object_set_new( visualization->getJsonMetaRoot(), "cell count", json_string( "Total numbers of cells" ) );
                 json_object_set_new( visualization->getJsonMetaRoot(), "particle count", json_string( "Total numbers of particles" ) );
             }
-            if (visualization->init() != 0)
+            if (visualization->init(reconnect?RetryEverySend:ReturnAtError) != 0)
```

Comment:
try not to outsmart the reader of your code.
```C++
CommunicatorSetting communicatorBehaviour = reconnect ? RetryEverySend : ReturnAtError;
if (visualization->init( communicatorBehaviour ) != 0)
```

Also, this only exists in the `isaac` `dev` so far, please draft a new release soon, give it a new version and increase the requirements in `INSTALL.md` and `CMakeLists.txt` of PIConGPU before we merge.